packages/swapper/src/swappers/PortalsSwapper/utils/fetchPortalsTradeOrder.ts

```diff
@@ -9,14 +10,22 @@ type PortalsTradeOrderParams = {
   inputToken: string
   inputAmount: string
   outputToken: string
-  slippageTolerancePercentage: number
   // Technically optional, but we always want to use an affiliate addy
   partner: string
   feePercentage?: number
   // Technically optional, but we want to explicitly specify validate
   validate: boolean
   swapperConfig: SwapperConfig
-}
+} & (
+  | {
+      slippageTolerancePercentage: number
+      autoSlippage?: never
+    }
+  | {
+      slippageTolerancePercentage?: never
+      autoSlippage: true
```

Comment:
Brings back auto-slippage as an explicit `autoSlippage` property, to ensure we never get a quote with auto-slippage when we shouldn't (i.e, only in case of upstream failures to get some expected slippage)